src/views/addEmergencyContact.js

```diff
@@ -82,88 +83,132 @@ const NumberSubForm = ({ i, values, errors, handleChange }) => {
     );
 };
 
-const AddEmergencyContact = () => {
+const emptyContact = {
+    name: "",
+    description: "",
+    contact_numbers: [{
+        number: "",
+        numtype: "",
+        extension: ""
+    }],
+};
+
+const AddEmergencyContact = (props) => {
+    const history = useHistory();
+    const userContext = useContext(UserContext);
+    const [initialized, setInitialized] = useState(false);
+    const [loading, setLoading] = useState(false);
+    const [editMode, setEditMode] = useState(false);
+    const [contactValues, setContactValues] = useState(emptyContact);
+
+    useMountEffect(() => {
+        const id = props.match.params.id;
+        if(!id && id !== 0) {
+            setInitialized(true);
+            return;
+        }
+        
+        setEditMode(true);
+        axios
+            .get(`${process.env.REACT_APP_API_URL}/emergencycontacts/${id}`, makeAuthHeaders(userContext))
```

Comment:
Note the axios requests will need to look like 
```
axios.get(`/api/emergencycontacts/${id}`, ...)
```
(see https://github.com/codeforpdx/dwellingly-app/pull/177)

Reply:
Thank you for pointing this out. I rebased on development for both my front end PR and my backend PR and made this change as recommended. Everything should now be forward-compatible and deployment-ready.